src/commands/wizard/lwcGenerationCommand.ts

```diff
@@ -22,9 +35,81 @@ export class LwcGenerationCommand {
                             panel.dispose();
                             return resolve();
                         }
+                    },
+                    {
+                        type: 'getQuickActionStatus',
+                        action: async (_panel, data, callback) => {
```

Comment:
Nit: `data` will probably go to `_data` too, unless you anticipate passing any in.